src/reducers/profile-view.js

```diff
@@ -206,15 +183,22 @@ function viewOptionsPerThread(state: ThreadViewOptions[] = [], action: Action) {
     }
     case 'ADD_TRANSFORM_TO_STACK': {
       const { threadIndex, transform } = action;
-      const expandedCallNodePaths = state[
-        threadIndex
-      ].expandedCallNodePaths.map(path =>
-        callNodePathAfterNewTransform(path, transform)
-      );
-      const selectedCallNodePath = callNodePathAfterNewTransform(
+      const expandedCallNodePaths = state[threadIndex].expandedCallNodePaths
+        .map(path => Transforms.applyTransformToCallNodePath(path, transform))
+        .filter(path => path.length > 0)
+        .filter((path, index, list) => {
+          // Ensure the path is unique and doesn't already exist in the list.
```

Comment:
Ah, no, that wouldn't work; `uniq` wants the comparator to return a number (and zero if the elements are equal).